### ISSUE TITLE
counts column format

### ISSUE THREAD

**Issue by xmpthz** [Illumina/ExpansionHunterDenovo#30]
Hi

First thank you for creating this tool and making it available.
I have run both outlier and control-case analysis on whole genomes and I have difficulties interpreting the results, in particular the "counts" column for 2 reasons:
- for the outlier analysis I have counts but not the sample names, unlike the results for the case-control approach
- some samples don't have counts. If I understood correctly it could be because the expansion is smaller than the read size and the repetition can't be quantified, but it could also be because there is not enough coverage or the motif is not present. Is there a way to differentiate these situations, and do you think more informations could be added to the counts to help with the interpretation ?

I hope you can help, thank you for your attention

**Comment by egor-dolzhenko** [comment]
Thank you very much for the feedback and apologies for not making the output easier to interpret. We will do our best to make the output more intuitive.

We can add a program option to EHdn to output (a) names of all samples and (b) counts for all samples, even if they are zeros.

And yes, your understanding is completely correct: Currently, the program does not report samples with zero counts of in-repeat reads and there could be multiple reasons for getting a count of zero (like low coverage and short repeat length). At the moment, EHdn does not check why the count is zero.

@mfbennett and I are currently working on a method that would perform a follow-up analysis of loci identified by EHdn. This way we can keep EHdn lightweight but provide users with an option to perform computationally-intensive analysis of a select subset of repeats. If this sounds useful and you are interested in testing out this approach, please feel free to reach out to me by email: edolzhenko@illumina.com. We could send you an initial version of this program as soon as it's ready.

Best wishes,
Egor

**Comment by xmpthz** [comment]
> We can add a program option to EHdn to output (a) names of all samples and (b) counts for all samples, even if they are zeros.

> @mfbennett and I are currently working on a method that would perform a follow-up analysis of loci identified by EHdn. This way we can keep EHdn lightweight but provide users with an option to perform computationally-intensive analysis of a select subset of repeats. If this sounds useful and you are interested in testing out this approach, please feel free to reach out to me by email: [edolzhenko@illumina.com](mailto:edolzhenko@illumina.com). We could send you an initial version of this program as soon as it's ready.

Yes I think it would be useful, I'm interested in testing and will send you an email.

Thank you for your help and quick response,
